debezium-assembly-descriptors/src/main/resources/assemblies/connector-distribution-no-drivers.xml

```diff
@@ -56,5 +56,13 @@
       </includes>
       <useDefaultExcludes>true</useDefaultExcludes>
     </fileSet>
+    <fileSet>
+      <directory>${project.build.directory}/generated-sources</directory>
```

Comment:
This can be removed here and in the other occurrences since you changed _OpenApiGeneratorMojo.java_.